frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/MainActivity.java

```diff
@@ -615,39 +657,4 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
         }
     }
 
-    // The app cannot currently handle the back button action to allow users
-    // to move between screens back and forth. What happens is the app is "moved"
-    // to background as if "home" button were pressed.
-    // To avoid unexpected behaviour, we prompt users and force the app process
-    // to exit which helps with preserving phone's resources by shutting down
-    // all goroutines.
-    //
-    // Without forced app process exit, some goroutines may remain active even after
-    // the app resumption at which point new copies of goroutines are spun up.
-    // Note that this is different from users tapping on "home" button or switching
-    // to another app and then back, in which case no extra goroutines are created.
-    //
-    // A proper fix is to make the backend process run in a separate system thread.
-    // Until such solution is implemented, forced app exit seems most appropriate.
-    //
-    // See the following for details about task and activity stacks:
-    // https://developer.android.com/guide/components/activities/tasks-and-back-stack
-    @Override
-    public void onBackPressed() {
```

Comment:
> API level 33.
Use [OnBackInvokedCallback](https://developer.android.com/reference/android/window/OnBackInvokedCallback) or androidx.activity.OnBackPressedCallback to handle back navigation instead.


https://developer.android.com/reference/android/app/Activity#onBackPressed()